src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentParty.kt

```diff
@@ -0,0 +1,88 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+import jakarta.persistence.EmbeddedId
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
+import java.io.Serializable
+import java.time.LocalDateTime
+
+@Embeddable
+class IncidentPartyId(
+  @Column(name = "INCIDENT_CASE_ID", nullable = false)
+  var incidentCaseId: Long,
+
+  @Column(name = "PARTY_SEQ", nullable = false)
+  var partySequence: Int,
+) : Serializable
+
+@Entity
+@Table(name = "INCIDENT_CASE_PARTIES")
+class IncidentParty(
+
+  @EmbeddedId
+  val id: IncidentPartyId,
+
+  @ManyToOne(fetch = FetchType.LAZY, optional = false)
+  @JoinColumn(name = "INCIDENT_CASE_ID", insertable = false, updatable = false)
+  val incident: Incident,
+
+  // Combination of Reference Codes: Staff role code=IR_STF_PART and Offender role code = IR_OFF_PART
```

Comment:
Because this is a combination of both Staff and Offender role codes, it means that the referenceCode can't be easily mapped.  Currently investigating if a hierarchical "Party" - one for offender and one for staff would mean I could separate them out.